source/developer-guides/debug.rst

```diff
@@ -0,0 +1,112 @@
+.. _debug:
+
+Source Level Debugging on Serial Port
+--------------
+
+This explains how to perform basic debugging operation with Intel UDK Debugger on SBL over serial port.
+
+**Prerequisite**
+
+Install Intel UDK Debugger on HOST::
+Intel UDK Debugger can be downloaded here for Windows and Linux, and just install it with default configuration values.
+
+https://firmware.intel.com/develop/intel-uefi-tools-and-utilities/intel-uefi-development-kit-debugger-tool#overlay-context=develop
```

Comment:
I think this link should be inlined in the sentence above. e.g. `[downloaded here](link here)`